.github/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+**mASAPP CI PR Template**
+
+### :bust_in_silhouette: Who
+
+_Who are you? If you are not Álvaro... Thanks and welcome! Could you explain a little bit who you are?
+
+
+### :point_up: Goal(s)
+
+_Short description of the changes added_ 
+
+
+### Resolved issues:**
+
+* [Tittle of the task 1 in Trello](https://trello.com/)
+...
+* [Tittle of the task N in Trello](https://trello.com/)
+
+
+### :ballot_box_with_check: Checklist
+
+- [ ] I've verified that unitary tests has been developed if it applies 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/manual/alopezna5/mASAPP_CI/pullRequest?prid=5264864)